pkgs/development/python-modules/cvxopt/default.nix

```diff
@@ -0,0 +1,78 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, isPyPy
+, python
+, openblasCompat # build segfaults with regular openblas
+, suitesparse
+, glpk ? null
+, gsl ? null
+, fftw ? null
+, withGlpk ? true
+, withGsl ? true
+, withFftw ? true
+}:
+
+buildPythonPackage rec {
+  pname = "cvxopt";
+  version = "1.1.9";
+
+  disabled = isPyPy; # hangs at [translation:info]
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0gcqq8ymjxv8qa5ss2pdhsj5bahvdxa6h2rlpp51520mjxrpw5cg";
+  };
+
+  buildInputs = [
+    openblasCompat
+    suitesparse
+  ] ++ lib.optionals withGlpk [
+    glpk
+  ] ++ lib.optionals withGsl [
+    gsl
+  ];
```

Comment:
You should be able to leave `buildInputs` out completely because they are all mentioned in `preConfigure`.